readme.md

```diff
@@ -0,0 +1,69 @@
+# Frontend Cookbook
```

Comment:
```suggestion
# Wolox Cookbook
```

Reply:
We could add a link to the cookbook
https://cookbook.wolox-fearmy.now.sh/buttons